applications/DeepSpeed-Chat/training/utils/perf.py

```diff
@@ -10,33 +10,26 @@
 def print_throughput(hf_model, args, e2e_time, rank=0):
     if rank <= 0:
         hf_config = hf_model.config
-        num_layers = getattr(hf_config, "num_hidden_layers",
-                             getattr(hf_config, "n_layer", None))
-        hidden_size = getattr(hf_config, "hidden_size",
-                              getattr(hf_config, "n_embd", None))
-        vocab_size = getattr(hf_config, "vocab_size", None)
-        assert all(
-            (num_layers, hidden_size, vocab_size)
-        ), "Could not determine number of layers, hidden size, and vocab size of the model"
+        num_layers, hidden_size, vocab_size = get_hf_configs(hf_config)
 
         gpus_per_model = torch.distributed.get_world_size()
         seq_length = args.max_seq_len
         batch_size = args.per_device_train_batch_size
         samples_per_second = batch_size / e2e_time
         checkpoint_activations_factor = 4 if args.gradient_checkpointing else 3
+        if args.lora_dim > 0:
+            k = args.lora_dim * 2 / hidden_size
```

Comment:
Nit: perhaps we can use a more descriptive name here?

Reply:
there is no name. k is just a factor that Zhewei and I discussed and made to address presence of lora for calculation gradient checkpointing factor.